cs41_final.sh

```diff
@@ -59,6 +83,91 @@ good_bye(){
   echo -e "${MENU}***************** ${RED_TEXT}GOOD BYE ${MENU}******************${NORMAL}"
   echo -e "${MENU}*********************************************${NORMAL}"
 }
+
+#----- Option 1 -----
+currentUsers(){
+  echo "Currently Logged-In Users:"
+  users
```

Comment:
this displays all users on one line, and duplicates users

To make this useful, you'll want to: 
- break them up into multiple lines
- remove duplicates